data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -741,6 +750,15 @@ def build_filing_json_meta_data(raw_filing_type: str, filing_type: str, filing_s
                 'fromLegalName': old_corp_name,
                 'toLegalName': new_corp_name,
             }
+    elif filing_type == 'putBackOff':
+        if (event_file_type := data['event_file_type']) == 'SYSDL_NULL':
+            filing_json['filing']['putBackOff'] = {
+                'details': 'Put back off filing due to expired limited restoration.'
+            }
+            meta_data['putBackOff'] = {
+                'reason': 'Limited Restoration Expired',
```

Comment:
while adding `FILE_AM_PF` and `FILE_CO_PF` we can add reason respectively

Reply:
tbh, [the current implementation](https://github.com/bcgov/lear/blob/release-2.141.0/legal-api/src/legal_api/reports/business_document.py#L481-L489) in release branch looks a bit hacky in my mind. And I prefer having a default name for put back off in a generic way. So I'm going to wait until we can merge both changes from release branch and migration branch together back to main.